src/screens/UserPortal/Pledges/Pledges.module.css

```
@@ -53,3 +53,89 @@
+}
+
+.editButton:is(:hover, :active) {
+  background-color: var(--color-blue-300);
+  color: var(--color-gray-700);
+  border-color: transparent;
+  box-shadow: none;
+}
+
+.progressPie {
+  --pie-fill-color: var(--color-blue-500);
+  width: var(--space-8);
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)